ingest.go

```diff
@@ -104,7 +105,7 @@ func ingestError(root xmlNode) Error {
 
 func duration(t string) time.Duration {
 	// Check if there was a valid decimal value
-	if s, err := strconv.ParseFloat(t, 64); err == nil {
+	if s, err := strconv.ParseFloat(strings.ReplaceAll(t, ",", ""), 64); err == nil {
```

Comment:
This will fail (or specifically return a default value of zero) if the value of `t` is something like `1,234.56s`. Moving the `strings.ReplaceAll` call to the top would fix:

```go
	// Remove comma characters from large numbers. 
	t = strings.ReplaceAll(t, ",", "")

	// Check if there was a valid decimal value
	if s, err := strconv.ParseFloat(t, 64); err == nil {
        ...
```

Reply:
Done, thanks!